src/frontend/next/src/components/SnackBar/index.ts

```diff
@@ -0,0 +1,3 @@
+import CustomizedSnackBar from './SnackBar';
+
+export default CustomizedSnackBar;
```

Comment:
I'm going to file a follow-up to remove all these `index.ts` files and just export the component directly.  This is fine for now.

Reply:
https://github.com/Seneca-CDOT/telescope/issues/1554

Reply:
@humphd : I'm going to remove `index.ts` as well in this PR to minimize the amount of works/changes later on.

Reply:
@humphd : I decided to keep the folder because some other components would need to have folder to keep all dependent components of that component. How you think?